### ISSUE TITLE
Cannot convert type System.Int64 using Docker.DotNet.Models.BoolQueryStringConverter.

### ISSUE THREAD

**Issue by MaitreDede** [ahmetalpbalkan/Docker.DotNet#17]
Hi,

I am using package nuget version 1.1.1.
Code :
```
 //Connect
            DockerClientConfiguration clientConfig = new DockerClientConfiguration(new Uri("http://ubuntu-docker.cloudapp.net:4243"));
            DockerClient client = clientConfig.CreateClient(new Version(1, 14));

            //List containers
            IList<ContainerListResponse> containers = client.Containers.ListContainersAsync(
    new ListContainersParameters()
    {
        Limit = 10,
    }).Result;
```
I have the exception :
Cannot convert type System.Int64 using Docker.DotNet.Models.BoolQueryStringConverter.

**Comment by ahmetalpbalkan** [comment]
This is fixed in #16. Will be in the next release.

**Comment by rjperes** [comment]
When can we expect a new NuGet package?
Thanks!

**Comment by ahmetalpbalkan** [comment]
@rjperes Hi, my bad. I totally forgot about this. Let me try to publish something tomorrow, if I forget again, please poke me again.

**Comment by rjperes** [comment]
Thanks! Keep up the good work! ;-)

**Comment by ahmetalpbalkan** [comment]
@rjperes I just released 1.1.2 on NuGet. 

https://www.nuget.org/packages/Docker.DotNet/1.1.2
https://github.com/ahmetalpbalkan/Docker.DotNet/releases/tag/1.1.2

Please try it out and let us know if you encounter any issues.

**Comment by rjperes** [comment]
Thanks, but there's a problem with the NuGet packages: they do not include any DLL, just a file named "_._" inside the "Docker.DotNet.1.1.2\lib\portable-net45+sl5+wp8+win8" folder!
